front/app/components/ProjectCard/index.tsx

```diff
@@ -603,7 +603,10 @@ const ProjectCard = memo<Props>(
         ctaMessage = <FormattedMessage {...messages.allocateYourBudget} />;
       } else if (participationMethod === 'information') {
         ctaMessage = <FormattedMessage {...messages.learnMore} />;
-      } else if (participationMethod === 'survey') {
+      } else if (
+        participationMethod === 'survey' ||
+        participationMethod === 'native_survey'
+      ) {
```

Comment:
This file is another huge one with lots of conditionals based on participation method. A big refactor would be nice here, but too big a lift for this PR..